integration/kubernetes/helm-chart/alluxio/templates/_helpers.tpl

```diff
@@ -2,8 +2,12 @@
 {{/*
 Expand the name of the chart.
 */}}
+{{/*
+Bring the items back
+*/}}
+
 {{- define "alluxio.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
```

Comment:
Please ignore the change in this file for now. (sth with template doesn't work in loops in other template files that refer here, so i disabled some logic.) This is will be reverted. This file will not be changed.